CANTroller2/src/globals.h

```diff
@@ -426,33 +408,16 @@ float gas_pulse_ccw_max_us = 2500;  // Servo ccw limit pulsewidth. Hotrc control
 float gas_pulse_park_slack_us = 30;  // Gas pulsewidth beyond gas_pulse_idle_us where to park the servo out of the way so we can drive manually (in us)
 
 // tachometer related
```

Comment:
Consider moving these to the Tachometer class and adding setters/getters.

Reply:
These are actually part of the gas servo, rather than the sensor (I have another change in the works to convert that as well). In the end we should probably have a Throttle or ThrottleManager class that contains the inputs, outputs, and pid logic for the throttle. but that's further down the line.